### PR TITLE
changes to the application process

### LINKED ISSUE #878: qard-hasan
Changing Copy

### PR DESCRIPTION
**Pull request by Arfah**
#closes #878 (I've changed some of the copy to make the barrier to entry a bit more easier to begin with)

### CODE REVIEW

File: data/qardhasan.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
 - title: What is the process?
-  blurb: Learning to code with Makers Academy is a serious commitment. To be considered you must complete the online resource Codecademy Ruby. You’ll need to fill in the form and record a video under 3 minutes on why you should receive this interest free loan. If selected you will be given more preparation work for the interview and asked to interview with us. We will then select a handful of people to receive the remote interest free loan.
+  blurb: Learning to code with Makers Academy is a serious commitment. You’ll need to fill in the form below and will be contacted and potentially invited for an interview. Give yourself the best chance at the interview and start coding with these <a href='http://help.makersacademy.com/knowledge_base/topics/what-can-i-do-to-improve-my-chances-of-getting-in' target='_blank'>resources</a>. If you pass the interview, there may be further steps to the process. Remember we only have a handful of fellowships to give away!
```
Comment (nikeshashar) [inline data/qardhasan.yml]: I'd be more specific here. After the application we'll defo ask them to undertake coding work *before* any interview. Ie they hit apply, we reply to them with thanks for your application, now please do 

* Codecademy
* Chris Pine
* Repl.it challenge

Comment (nikeshashar) [inline data/qardhasan.yml]: Basically it's merit based. So they have to be showing some sort of desire to code themselves.

Comment (Arfah) [inline data/qardhasan.yml]: so what do you suggest I put? It links to the resource in the help centre.

Comment (Arfah) [inline data/qardhasan.yml]: I agree it should be but I think there is a danger of shutting people out from the start.  I'd rather speak to them once the application is submitted and I ask those questions on the Typeform.

**Comment by nikeshashar** [comment]
:shipit:

**Pull request merged**